src/components/Tabs/README.md

```diff
@@ -183,3 +183,50 @@ Also known as [Segmented controls](https://developer.apple.com/design/human-inte
 ![Fixed tabs on iOS](/public_images/components/Tabs/ios/fixed@2x.png)
 
 <!-- /content-for -->
+
+### Tabs with badge content
+
+Use to inform a piece of information about the tabs.
```

Comment:
I'm not sure about this example, it sounds like we're defining a UX pattern (which I'm not sure has been validated by the Polaris team). 

We could probably make the example a little bit more generic (e.g. passing down bolded text?) just to exemplify that you can pass any React node. How are other similar components that allow for any React node exemplified?

Reply:
> I'm not sure has been validated by the Polaris team

The polaris team isn't here to be some overarching judge who says what you can and can't do. Us "validating" things that product teams want to do is basically saying "the company doesn't trust product teams, then need some oversight" which is a straight up overzealous and untrue nonsense :) We trust product teams, you don't need our "blessing" - you can get a UX review from anybody, not just polaris - it's just that we're usually a bit more readily available for that sort of review work.

---

I'm happy with the description test, though I wonder if a more realistic example might be to show the number of items in that will be shown within the tab, something like:

- `All <Badge status="info">10+</Badge>`
-  `Accepts marketing <Badge status="info">5</Badge>`


What do ya think?

Reply:
Really appreciate the context, Ben! 🙏

Reply:
Agreed Ben, the number of items is definitely more realistic! Will update.